### DISCUSSION TITLE
How can I do a wildcard route?

### DISCUSSION THREAD

**Discussion by justinmchase** [oakserver/oak#464] [Q&A] [answered]
```
router.get('/assets/*', ...)
```

Seems to not work, how can I specify a route which gets all requests under `/assets`. The examples only show `/assets/:id` but I need all nested paths.

**Comment by kitsonk** [comment, marked as answer]
@justinmchase it uses [path_to_regex](https://github.com/pillarjs/path-to-regexp) underneath, so anything that works there would work here.  There is also an online [tester](https://forbeslindesay.github.io/express-route-tester/) for it.

For your specific use case you want:

`/asset/:id+` or `/asset/:id*`

**Reply by justinmchase** [reply]
Ah, perfect thanks.

It may be worth putting more prominently in the docs, I really struggled to find it or any examples.
